hack/kind-install-for-capd.sh

```diff
@@ -26,7 +26,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-NAME="capi-test"
+if [[ -z "$CAPI_KIND_CLUSTER_NAME" ]]; then
+  NAME="capi-test"
+else
+  NAME="${CAPI_KIND_CLUSTER_NAME}"
+fi
```

Comment:
Q: Should we drop L35-L44 so there is only a single-mode the set the cluster name (envvar vs  envvar+command arg)?
@killianmuldoon opinions?

Reply:
Definitely - using and env variable will make it easy to use the same value elsewhere (e.g. tilt setup) where needed. @DiptoChakrabarty could you remove the while block from L35-L44?

Reply:
okay sure